src/Hadolint/Rules.hs

```diff
@@ -226,7 +226,7 @@ noLatestTag = instructionRule code severity message check
     message =
         "Using latest is prone to errors if the image will ever update. Pin the version explicitly \
         \to a release tag."
-    check (From (TaggedImage _ tag)) =
+    check (From (TaggedImage _ tag _)) =
```

Comment:
I believe that tag and name are now nicely separated, we can replace this _hack_ by 

```haskell
check (From (TaggedImage _ tag _)) = tag /= "latest"
```

Reply:
ah, good catch